dnf5/context.cpp

```diff
@@ -176,6 +176,8 @@ namespace {
 
 class RpmTransCB : public libdnf::rpm::TransactionCallbacks {
 public:
+    RpmTransCB() { multi_progress_bar.set_total_bar_visible_limit(static_cast<std::size_t>(-1)); }
```

Comment:
This is kinda not transparent at the first sight, it took me some time to understand what it is actually doing and that -1 is transformed to `SIZE_MAX`... Could you please define some local constant for this purpose like `NEVER_VISIBLE_LIMIT` or something like that?

Reply:
> This is kinda not transparent at the first sight, it took me some time to understand what it is actually doing and that -1 is transformed to SIZE_MAX

Yes. It is a portable method to find the maximum value of `std::size_t`. It is guaranteed to work with any unsigned type. But I agree that it is more transparent and user-friendly to define a constant for this purpose.